mozilla/foundation.mozilla.org
# Updated weights for quote blocks and callouts

Closes #7287

**Link to sample test page**:
Link to updated style guide: https://foundation-s-7287-quote-9aj1mz.herokuapp.com/en/style-guide/


**Steps to test**:

1. Visit [the updated style guide](https://foundation-s-7287-quote-9aj1mz.herokuapp.com/en/style-guide/) and scroll to the bottom to take a look at the Callout and Quoteblock component.
2. If both appear OK with the new font weights, testing is done! 


## Checklist

**Changes in Models:**
